securedrop/sdconfig.py

```diff
@@ -11,154 +14,216 @@
     # http://flake8.pycqa.org/en/latest/user/error-codes.html?highlight=f401
     from typing import List, Dict  # noqa: F401
 
+CONFIG_DIR = '/etc/securedrop'
+
+
+class _FlaskConfig(object):
+
+    DEBUG = False
+    TESTING = False
+    WTF_CSRF_ENABLED = True
+
+    def __init__(self, secret_key, cookie_name, env):
+        # type: (str, str, str) -> None
+        self.SECRET_KEY = secret_key
+        self.SESSION_COOKIE_NAME = cookie_name
+
+        if env == 'prod':
+            # This is recommended for performance, and also resolves #369
+            self.USE_X_SENDFILE = True  # type: ignore
```

Comment:
Can we please have all the possible attributes of the class with some default values? Let us say, locked down production values, and only enable/set in the other cases. Will really help to read/type annotations.

Reply:
Flask handles the default on its own, and I don't think we should set them ourselves.

Reply:
@heartsucker I am asking defaults to enable/use `mypy` in a better way.